### PR TITLE
28 prototype expo secure store for key did

### LINKED ISSUE #32: THIS IS ONLY A TEST ISSUE
Prototype saving passphrase and did from our test page using expo secure store.

### PR DESCRIPTION
**Pull request by 2byrds**
Switched from old branch referenced in https://github.com/roots-id/rootswallet/pull/28 to this PR.
closes #32

### CODE REVIEW

**Comment by 2byrds** [comment]
Note: the name is misleading.  it saves the DID by passphrase, not key.  will tackle the key in a new issue.

File: App.tsx [hunk 1]
```diff
@@ -1,27 +1,75 @@
...
+        maxLength={41}
+        clearTextOnFocus
+        multiline={false}
+        editable={false}
+        value={did}
+      />
+      <Button
+        title={t('Home.SavePassDID')}
+        onPress={() => {
+          savePassDID(passphrase, did);
```
Comment (Essbante) [inline App.tsx:53]: Are you using the key derivation binary seed passphrase also as a key for the SecureStore?

File: App.tsx [hunk 2]
```diff
@@ -1,27 +1,75 @@
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View, Button, NativeModules } from 'react-native';
-import { useTranslation } from 'react-i18next'
-import './localization'
+import { StyleSheet, Text, TextInput, View, Button, NativeModules } from 'react-native';
+import { useTranslation } from 'react-i18next';
+import './localization';
+import { React, useState } from 'react';
+import * as SecureStore from 'expo-secure-store';
```
Comment (Essbante) [inline App.tsx:6]: SecureStore has a Size limit for a value of 2048 bytes. is that enough?

**Review by Essbante — COMMENTED** [review]
Let's schedule a peer review session,

**Comment by Essbante** [comment]
It may be interesting to explore secureOptions->requireAuthentication: https://docs.expo.dev/versions/latest/sdk/securestore/#securestoreoptions

**Pull request closed without merging**